roster/tests.py

```diff
@@ -108,35 +108,25 @@ def test_delinquency(self) -> None:
         alice: Student = StudentFactory.create(semester=semester)
 
         self.assertEqual(alice.payment_status, 0)  # because no invoice exists
-        invoice: Invoice = InvoiceFactory.create(
-            student=alice,
-            preps_taught=2,
-        )
+        with freeze_time("2022-08-05", tz_offset=0):
+            invoice: Invoice = InvoiceFactory.create(
+                student=alice,
+                preps_taught=2,
+            )
 
         # Alice has paid $0 so far
         self.assertEqual(invoice.total_owed, 480)
         with freeze_time("2022-09-05", tz_offset=0):
             self.assertEqual(alice.payment_status, 4)
-            self.assertFalse(alice.is_payment_locked)
```

Comment:
is there a reason the `alice.is_payment_locked` tests were deleted?

Reply:
I merged `is_payment_locked` and `is_delinquent` in this pr. I can undo if desired

Reply:
I guess that's fine